lib/kennel/github_reporter.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 module Kennel
   class GithubReporter
+    MAX_COMMENT_SIZE = 65536
+    TRUNCATED_MSG = "\n```\n... (truncated)" # finish the code block so it look nice
```

Comment:
Not sure if this makes sense for errors but anyway, they're unlikely to be that long.